creating-virtual-machines/virtualized-hardware-configuration.adoc

```diff
@@ -471,3 +471,63 @@ Limitations
 * a node must have pre-allocated hugepages
 * hugepages size cannot be bigger than requested memory
 * requested memory must be divisible by hugepages size
+
+Input Devices
+~~~~~~~~~~~~~
+
+Tablet
+++++++
+
+Kubevirt supports input devices. The only type which is supported is `tablet`. 
+Tablet input device supports only `virtio` and `usb` bus. Bus can be empty. In that case,
+`usb` will be selected.
+
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: myvm
+spec:
+  domain:
+    devices:
+      inputs:
+      - type: tablet
+        bus: virtio
+        name: tablet1
+      disks:
+      - name: myimage
+        disk: {}
+  volumes:
+    - name: myimage
+      persistentVolumeClaim:
+        claimname: myclaim
+----
+
+USB Devices
```

Comment:
That section seems to be pretty redundant. Otherwise it looks good.

Reply:
They are almost the same, because you cannot use usb bus anywhere else than in input device. And for demonstration how to configure usb device i had to insert the same yaml as is in input paragraph.

Reply:
Why do I need this extra information? I can see above that I can add a `usb` input device, which as you say, describes everything we support.

Reply:
I removed the usb device paragraph, as all informations are already in input device paragraph